diagnostic.md

```diff
@@ -21,13 +21,32 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts
+
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function getDonuts() {
+    debugger;
```

Comment:
Whoops - don't forget to take out any extraneous `debugger` statements when you're done.